github-sync/github-data/sigstore/repositories.yaml

```diff
@@ -1444,16 +1444,33 @@ repositories:
           - yamllint
           - test
           - lint
-          - validate
-          - client
         pushRestrictions:
+          - tuf-root-signing-codeowners
+          - sigstore-bot
+          - sigstore-review-bot
+        dismissalRestrictions:
           - tuf-root-signing-codeowners
           - sigstore-keyholders
```

Comment:
do you want to also remove keyholders here as well and for publish?

Reply:
Yeah why not: this may end up being the only reason to have a keyholders team and we might want to just remove the whole team to simplify things